pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -19,9 +19,9 @@ package tektonconfig
 import (
 	"context"
 	"fmt"
-	"math"
-	"regexp"
-
+	securityv1 "github.com/openshift/api/security/v1"
```

Comment:
can you please use `goimports`

this is validating that and same the case with many other files

Reply:
done